packages/create-bison-app/template/README.md.ejs

```diff
@@ -60,97 +61,212 @@ Bison uses [Nexus Schema](https://nexusjs.org/docs/) to create the GraphQL API.
+    resolve: async (_root, args, ctx) => {
+      const { where = {}, orderBy = [] } = args;
 
-// Mutatations
-export const OrganizationMutations = extendType({
-  type: 'Mutation',
-  definition(t) {
-    t.crud.createOneOrganization({
-      alias: 'createOrganization',
-      authorize: (_root, _args, ctx) => !!ctx.user,
-      computedInputs: {
-        users: ({ ctx }) => ({
-          connect: [{ id: ctx.user.id }],
-        }),
-      },
-    });
-  },
-});
-```
+      return await ctx.db.organization.findMany({ where, orderBy });
+    }
+  });
+
+  export const findUniqueOrganizationQuery = queryField('organization', {
+    type: 'Organization',
+    description: 'Returns a specific Organization',
+    authorize: (_root, _args, ctx) => !!ctx.user,
+    args: {
+      where: nonNull(arg({ type: 'OrganizationWhereUniqueInput' }))
+    },
+    resolve: (_root, args, ctx) => {
+      const { where } = args;
+      return ctx.prisma.organization.findUnique({ where })
+    },
+  });
+
+  // Mutations
+  export const createOrganizationMutation = mutationField('createOrganization', {
+    type: 'Organization',
+    description: 'Creates a Organization',
+    authorize: (_root, _args, ctx) => isAdmin(ctx.user),
+    args: {
+      data: nonNull(arg({ type: 'CreateOrganizationInput' })),
+    },
+    resolve: async (_root, args, ctx) => {
+      return await ctx.db.organization.create(args);
+    }
+  });
+
+  export const updateOrganizationMutation = mutationField('updateOrganization', {
+    type: 'Organization',
+    description: 'Updates a Organization',
+    authorize: (_root, _args, ctx) => isAdmin(ctx.user),
+    args: {
+      where: nonNull(arg({ type: 'OrganizationWhereUniqueInput'})),
+      data: nonNull(arg({ type: 'UpdateOrganizationInput' })),
+    },
+    resolve: async (_root, args, ctx) => {
+      const { where, data } = args;
+
+      return await ctx.db.organization.update({ where, data });
+    }
+  });
+
+  // Mutation Inputs
+  export const CreateOrganizationInput = inputObjectType({
+    name: 'CreateOrganizationInput',
+    description: 'Input used to create a organization',
+    definition: (t) => {
+      t.nonNull.string('name');
+    },
+  });
+
+  export const UpdateOrganizationInput = inputObjectType({
+    name: 'UpdateOrganizationInput',
+    description: 'Input used to update a organization',
+    definition: (t) => {
+      t.nonNull.string('name');
+    },
+  });
+
+  // Query Inputs
+  export const OrganizationOrderByInput = inputObjectType({
+    name: 'OrganizationOrderByInput',
+    description: 'Order organization by a specific field',
+    definition(t) {
+      t.field('name', { type: 'SortOrder' });
+    },
+  });
+
+  export const OrganizationWhereUniqueInput = inputObjectType({
+    name: 'OrganizationWhereUniqueInput',
+    description: 'Input to find organizations based on unique fields',
+    definition(t) {
+      t.id('id');
+      // add DB uniq fields here
```

Comment:
Great call out!

Reply:
I just copied and pasted these code examples so I can't take credit for them! Most of these examples are the result of our generators which were recently updated by @mthomps4.